Marlin/language_en.h

```diff
@@ -261,6 +261,24 @@
 #ifndef MSG_ESTEPS
 #define MSG_ESTEPS                          "Esteps/mm"
 #endif
+#ifndef MSG_X1OFFSET
+#define MSG_X1OFFSET                         "T1X Offset"
```

Comment:
I think we can say with confidence that the current quasi-standard has been defined by various firmware projects as well as Skeinforge. But the real question is, who should drive the standards and who should conform, the firmwares or the slicers? Maybe some sort of consortium should be formed that compose these standards, something like the Open Handset Alliance (there's currently no W3C equivalent for reprap).